sc-platform-api/src/documentation/article/create.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiDocumentationParams } from '../decorators';
+
+export const ArticleDocumentationCreate: ApiDocumentationParams = {
+  isAuth: true,
+  ok: true,
```

Comment:
In this endpoint response type is `ArticleResponse`
```suggestion
  ok: {
    type: ArticleResponse,
  },
```